### PR TITLE
Fix formatting on AWS docs

### PR DESCRIPTION
**Pull request by jonasrosland**
This should fix https://github.com/heptio/velero/issues/1494

Signed-off-by: Jonas Rosland <jrosland@vmware.com>

### CODE REVIEW

File: site/docs/master/aws-config.md [hunk 1]
```diff
@@ -196,113 +192,109 @@ For more complex installation needs, use either the Helm chart, or add `--dry-ru
...
-        name: velero
-    spec:
-        replicas: 1
-        template:
-            metadata:
-                labels:
-                    component: velero
-                annotations:
-                    iam.amazonaws.com/role: arn:aws:iam::<AWS_ACCOUNT_ID>:role/<VELERO_ROLE_NAME>
-    ...
```
Comment (skriss) [inline site/docs/master/aws-config.md]: I think the point with the ellipsis here was that we aren't showing the entire deployment manifest, just the relevant subsection for this step.

**Comment by skriss** [comment]
Preview site looks good, thanks for this @jonasrosland!

**Comment by skriss** [comment]
BTW, do you have an open issue somewhere to track adding constrast to the code blocks? If not, I'll create one in this repo.

**Comment by jonasrosland** [comment]
I do not have an open issue for adding contrast to the coding blocks. Please add one for that feature.
This is ready to merge to fix the formatting here so things don't break anymore, and I'll add more docs updates in another PR.

**Pull request merged**